shared-bindings/audioio/AudioOut.c

```diff
@@ -43,13 +43,15 @@
 //|
 //| AudioOut can be used to output an analog audio signal on a given pin.
 //|
-//| .. class:: AudioOut(left_channel, right_channel=None)
+//| .. class:: AudioOut(left_channel, *, right_channel=None, default_value=0x8000)
 //|
 //|   Create a AudioOut object associated with the given pin(s). This allows you to
 //|   play audio signals out on the given pin(s).
 //|
 //|   :param ~microcontroller.Pin left_channel: The pin to output the left channel to
 //|   :param ~microcontroller.Pin right_channel: The pin to output the right channel to
+//|   :param int default_value: The default output value. Samples should start and end with this
```

Comment:
I was trying to think of a more descriptive name for this than "default_value". Maybe "silence_value" or "midpoint value"?. @CedarGroveStudios, is there a standard name?